### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by nfelt**
This PR contains relnotes and a couple small bug fixes for our 1.7.0 release.

Please confirm the wording of the proposed release notes. I'd like to avoid making any further edits to them when merging them back into master's RELEASE.md.

cc @caisq

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,37 @@
+# Release 1.7.0
+
+The 1.7 minor series tracks TensorFlow 1.7.
+
+## Hightlights
+
+- (Beta) New Beholder plugin that shows a live video feed of tensor data during
```
Comment (jart) [inline RELEASE.md]: Please mention this requires a local disk and doesn't work on GCS, S3, etc.

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,37 @@
...
+  model training. Developed and contributed by Chris Anderson (@chrisranderson).
+  See for details:
+  https://github.com/tensorflow/tensorboard/tree/1.7/tensorboard/plugins/beholder
```
Comment (jart) [inline RELEASE.md]: Can this link be shortened to `[tensorboard/plugins/beholder](...)`?

File: RELEASE.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
+# Release 1.7.0
+
+The 1.7 minor series tracks TensorFlow 1.7.
+
+## Hightlights
+
+- (Beta) New Beholder plugin that shows a live video feed of tensor data during
+  model training. Developed and contributed by Chris Anderson (@chrisranderson).
```
Comment (jart) [inline RELEASE.md]: `Developed and contributed by Chris Anderson (@chrisranderson)` could be shortened to `by @chrisranderson`

File: RELEASE.md (hunk 1 above)
Comment (jart) [inline RELEASE.md]: It should be OK to remove `(Beta)`. It works. We nor Chris are able to do more than maintenance on Beholder at the moment.

File: RELEASE.md (hunk 3 above)
Comment (nfelt) [reply RELEASE.md]: That generates a relative link which doesn't work if viewing this file anywhere other than from the final location (e.g. it wouldn't work when viewed from this pull request page or from a commit view, etc.).  Also, this style of link was used for the last several dashboard intro blurbs further down in this file.

File: RELEASE.md (hunk 2 above)
Comment (nfelt) [reply RELEASE.md]: That generates a relative link which doesn't work if viewing this file anywhere other than from the final location (e.g. it wouldn't work when viewed from this pull request page or from a commit view, etc.).  Also, this style of link was used for the last several dashboard intro blurbs further down in this file.

File: RELEASE.md (hunk 1 above)
Comment (nfelt) [reply RELEASE.md]: That's why I prefer having the `(Beta)` label there actually.  I think it makes it clearer that it's not intended to be fully supported to the same extent as e.g. the scalars dashboard.  Also, while it works, there are certainly still robustness issues and edge cases (like the file permissions problem I just fixed) and I think it's worth making that clearer to users.

Comment (nfelt) [reply RELEASE.md]: Done.

File: RELEASE.md (hunk 3 above)
Comment (nfelt) [reply RELEASE.md]: Done.

File: RELEASE.md (hunk 1 above)
Comment (jart) [reply RELEASE.md]: [Optional] The wording 'New ...' might be enough to communicate it's not yet been battle-tested. We'll maintain Beholder going forward. That may involve help from the community.

**Pull request merged**